docs/articles/documentation/using-testcafe/common-concepts/live-mode.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: docs
+title: Live Mode
+permalink: /documentation/using-testcafe/common-concepts/live-mode.html
+---
+# Live Mode
+
+Live mode keeps the TestCafe process and browsers opened the whole time you are working on tests. Changes you make in code immediately restart the tests. That is, live mode allows you to see test results instantly.
+
+![Live mode demonstrated in action](../../../images/testcafe-live.gif)
+
+> This feature replaces the `testcafe-live` module. This module is now deprecated.
+
+## How to Enable Live Mode
+
+Use the [-L (--live)](../command-line-interface.md#-l---live) flag to enable live mode from the command line interface.
+
+```sh
+testcafe chrome tests/test.js -L
+```
+
+In the API, create a [live mode runner](../programming-interface/livemoderunner.md) with the [testcafe.createLiveModeRunner](../programming-interface/testcafe.md#createlivemoderunner) function and use it instead of a [regular test runner](../programming-interface/runner.md).
+
+```js
+const createTestCafe = require('testcafe');
+let testcafe         = null;
+
+createTestCafe('localhost', 1337, 1338)
+    .then(tc => {
+        testcafe         = tc;
+        const liveRunner = testcafe.createLiveModeRunner();
+
+        return liveRunner
+            .src('tests/test.js')
+            .browsers('chrome')
+            .run();
+    })
+    .then(() => {
+        testcafe.close();
+    });
+```
+
+## How Live Mode Works
+
+When you run tests with live mode enabled, TestCafe opens the browsers, run tests there, shows the reports and waits for your further actions.
+
+Then TestCafe starts watching for changes in the test files and all files that are referenced in them (like helper modules). Once you make changes in any of those files and save them, TestCafe immediately reruns the tests.
+
+When the tests are done, browsers stay on the last opened page so you can work with it and explore it with the browser's developer tools.
+
+> To work with a single test, use the [only](../../test-api/test-code-structure.md#skipping-tests) function.
+
+You can use live mode with any browsers: local, remote, mobile or headless.
+
+> Live mode is designed to work with tests locally. Do not use it in CI systems.
+
+### Console Commands in Live Mode
```

Comment:
Why the commands instead of shortcuts, IMO a console command supposes it should been submitted (press enter) its after all. Am I wrong?